Mobile/Android/Templates/471-reliable_uploads_actions_uploads_view.md

```diff
@@ -0,0 +1,64 @@
+### Uploads view
+
+#### Pr: https://github.com/owncloud/android/issues/471
+
+Devices:
+
+Server version:
+
+<br>
+ 
+Test Case ID | Test case     | Steps   | Expected behavior | Result | Related Comments
+|:----:|:------------- |:---------------|:-------------|-------------:|----------:
+|**Views**|||||
+|1| New option in menu | 1. Swipe to display the main menu <br> 2. Change orientation of device | 1. An new option "Uploads" is displayed in the menu<br> 2. The view is correct in both orientations|||
```

Comment:
Historical thing, when we first launch the iOS upload app, we were asked to add any information when there weren't any upload, what happens here?
https://github.com/owncloud/ios/blob/develop/Owncloud%20iOs%20Client/SupportingFiles/Languages/en.lproj/Localizable.strings#L179